src/global.d.ts

```diff
@@ -1,3 +1,36 @@
+export interface MarketBean {
```

Comment:
lol can tell you coded a lot in java. Can name this `MarketConfig` instead for consistency with how its name in web repo. Also the api name is /markets/configs so they match